relay/kusama/src/xcm_config.rs

```diff
@@ -189,7 +189,7 @@ pub struct XcmConfig;
 impl xcm_executor::Config for XcmConfig {
 	type RuntimeCall = RuntimeCall;
 	type XcmSender = XcmRouter;
-	type XcmRecorder = ();
+	type XcmRecorder = XcmPallet;
```

Comment:
maybe we could regenerate xcm weights also, this adds one more read :),
but also we can leave it to the next time